src/main.cpp

```diff
@@ -34,7 +34,7 @@
 SpaInterface si;
 
 
-Blinker led(LED_BUILTIN);
+Blinker led(LED_PIN);
```

Comment:
We really need to wrap this in a ifdef block, along with the relevant entry from loop()